Raygun4UWP.nuspec

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata minClientVersion="2.8.3">
+    <id>Raygun4UWP</id>
+    <title>Raygun4UWP</title>
+    <version>1.0.0</version>
+    <releaseNotes>
+    </releaseNotes>
+    <authors>Raygun</authors>
+    <copyright>Copyright 2019</copyright>
+    <iconUrl>https://app.raygun.com/Content/Images/nuget-icon.png</iconUrl>
+    <licenseUrl>https://raw.github.com/MindscapeHQ/raygun4uwp/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/MindscapeHQ/raygun4uwp</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Raygun provider for Universal Windows (UWP) applications</description>
+    <summary>Raygun provider for Universal Windows (UWP) applications</summary>
+  <dependencies>
+    <group>
+      <dependency id="Newtonsoft.Json" version="12.0.2" />

```

Comment:
The new Xamarin Forms provider has moved to Newtonsoft.Json and I think it's a good idea so have done the same here. Xamarin.Forms is using 11.0.2, is that low enough for what you were thinking, or do you have a recommended version?

Reply:
My thinking here is it forces consumers of raygun4uwp to use Newtonsoft.Json version 12.0.2 or above. This can be painful if it forces you to upgrade. I'd rather we had a dependency of the lowest possible version that we actually depend on. Given we're not using anything advanced, unless there is a bug fix we need in a specific version, we could probably go a lot lower.

Reply:
:+1: Found that the lowest possible version that works for this is 4.5.7 which I've switched to.